lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -26,18 +27,27 @@ enforceExistence.configure = function(options) {
         'jsDoc.enforceExistence rule was not configured properly'
     );
 
+    var optionsToPolicyMap = [
```

Comment:
;-) Yeah, these IDE integration are pretty queasy on paths and global installation. In my case it's sublime on mac/win using samba/fuse to home directory on a linux server, and there's a lot of magics ;-D

Not sure that this fix will work well in all cases. "plugins" itself should resolve this imho.